suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -70,7 +70,10 @@ const merge = (device: AcquiredDevice, upcoming: Partial<AcquiredDevice>): Trezo
 
 const getShouldUseEmptyPassphrase = (device: Device, deviceInstance?: number): boolean => {
     if (!device.features) return false;
-    if (isNative() && typeof deviceInstance === 'number' && deviceInstance === 1) {
+    if (
+        isNative() &&
+        ((typeof deviceInstance === 'number' && deviceInstance === 1) || !deviceInstance)
+    ) {
```

Comment:
Maybe it can be simplified like this?
```suggestion
    if (isNative() && (!deviceInstance || deviceInstance === 1)) {
```

and it could use some comment why `!deviceInstace` is added

Reply:
Good catch! Fixed and rebased